Jenkinsfile-Release

```diff
@@ -12,22 +12,59 @@
  */
 @Library(['private-pipeline-library', 'jenkins-shared', 'int-jenkins-shared']) _
 
-def jira = [
+final jira = [
     versionPrefix: '', project: 'INT', projectId: '12410',
     credentialId : 'jenkins-jira', autoRelease: true, failOnError: true
 ]
 
-def jiraVersionMappings = [
-    'nexus-iq-server':'helm-iq', 
-    'nexus-repository-manager':'helm-nxrm'
+final jiraVersionMappings = [
+  'nexus-iq-server': 'helm-iq',
+  'nexus-repository-manager': 'helm-nxrm',
 ]
 
+final chartLocation = [
+  'nexus-iq-server': 'nexus-iq',
+  'nexus-repository-manager': 'nexus-repository-manager',
+]
+
+properties([
+  parameters([
+    choice(
+      choices: ['', 'nexus-iq-server', 'nexus-repository-manager'],
+      name: 'chart',
+      description: 'Chart to deploy.',
+    ),
+    string(
+      name: 'appVersion',
+      description: 'Version of the application image, like "1.139.0"',
+    ),
+    string(
+      name: 'chartVersion',
+      description: '(Optional) Version of the Chart, like "139.0.0". If omitted, it will be calculated from the appVersion.',
+    ),
+  ])
+])
+
+final chartVersion = calculateChartVersion(params.chartVersion, params.appVersion)
+
 dockerizedBuildPipeline(
   prepare: {
+    if (! params.chart) {
+      error('Chart parameter is required.')
+    }
+    if (! params.appVersion) {
+      error('The appVersion is required.')
+    }
     githubStatusUpdate('pending')
   },
   buildAndTest: {
-    sh './build.sh'
+    final branch = gitBranch(env)
+    runSafely 'git config user.email "sonatype-ci@sonatype.com"'
```

Comment:
you can use this convenience mechanism now: https://docs.sonatype.com/display/CDI/sonatypeZionGitConfig